src/index.html

```diff
@@ -12,6 +12,176 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      >
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <div class="data">
+          <label>
+            Surname:
+            <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            class="data"
+            required
+            >
+          </label>
+        </div>
+        <div class="data">
+          <label>
+            Name:
+            <input
+              type="text"
+              name="name"
+              autocmplete="off"
+              required
+              >
```

Comment:
Refer to the screenshot again and fix remaining inputs
https://mateacademyen.slack.com/archives/C03S1JURNG5/p1660665961358809
```suggestion
            <input
              type="text"
              name="name"
              autocmplete="off"
              required
            >
```